functions/logStripeEvent.js

```diff
@@ -0,0 +1,90 @@
+const functions = require('firebase-functions');
+const admin = require('firebase-admin');
+const logDonation = require('./logDonation');
+if (admin.apps.length === 0) {
+  admin.initializeApp();
+}
+
+const DEFAULTCPL = 1.0;
```

Comment:
In the future we should move this to a global config since it's being used in multiple areas in the app

Reply:
agreed, we should discuss the right way to set up global configs because a lot of variables in the codebase could be moved there